components/layout/query.rs

```diff
@@ -463,16 +463,16 @@ impl FragmentBorderBoxIterator for UnioningFragmentScrollAreaIterator {
         let bottom_padding = (border_box.size.height.to_f32_px() - bottom_border - top_border) as i32;
```

Comment:
This looks somewhat fishy to me, in the sense that `margin` accounts for writing-mode, but not `padding`... Not that it wasn't wrong before, though...

Reply:
Are you saying `bottom_padding` here is weird? What is `writing-mode`?